### ISSUE TITLE
Espresso regression tests failing sporadically

### ISSUE THREAD

**Issue by seadowg** [getodk/collect#3205]
The `DrawWidgetTest` and `SignatureWidgetTest` appear to be failing on most runs on Test Lab but not in a predictable manner. The actual failure appears to be:

```
androidx.test.espresso.NoMatchingViewException: No views in hierarchy found matching: with id: org.odk.collect.android:id/jumpEndButton 
```

**Comment by grzesiek2010** [comment]
I'm pretty sure the problem is related to orientation changes...
in those tests, we use DrawActivity which works in landscape mode. When a user returns from that activity to DrawWidget/SignatureWidget a rotation will take place landscape->portrait. It might take some time and probably it's the cause.

What do you think about calling [waitForIdleSync](https://developer.android.com/reference/android/app/Instrumentation.html#waitForIdleSync()) every time we get back from that activity to wait for the rotation end?

https://stackoverflow.com/a/38860384

**Comment by seadowg** [comment]
@grzesiek2010 my impression is that Espresso should be doing that for us but it's definitely worth a try. I hadn't thought about the screen rotations and it wouldn't surprise me if that messes with Espresso...

**Comment by grzesiek2010** [comment]
@seadowg so maybe let's try it https://github.com/opendatakit/collect/pull/3277

**Comment by seadowg** [comment]
@grzesiek2010 do you think it's worth merging in #3276 first as a temporary measure or are you pretty close with #3277?

**Comment by seadowg** [comment]
Sorry I should have closed this with #3276.
